### ISSUE TITLE
API 31 support

### ISSUE THREAD

**Issue by rlatapy-luna** [snapcall/snapcall_sdk_android#2]
• Crash due to pending intent flags
https://developer.android.com/about/versions/12/behavior-changes-12#pending-intent-mutability
```
2021-11-08 10:34:22.438 _._._._ E/AndroidRuntime: FATAL EXCEPTION: main
    Process: io.snapcall.snapcall_android_framework, PID: 27613
    java.lang.RuntimeException: Unable to start service io.snapcall.snapcall_android_framework.VertoSignalingHandler@60d51e6 with Intent { cmp=com.prismea.android.dev/io.snapcall.snapcall_android_framework.VertoSignalingHandler (has extras) }: java.lang.IllegalArgumentException: com.prismea.android.dev: Targeting S+ (version 31 and above) requires that one of FLAG_IMMUTABLE or FLAG_MUTABLE be specified when creating a PendingIntent.
    Strongly consider using FLAG_IMMUTABLE, only use FLAG_MUTABLE if some functionality depends on the PendingIntent being mutable, e.g. if it needs to be used with inline replies or bubbles.
        at android.app.ActivityThread.handleServiceArgs(ActivityThread.java:4661)
        at android.app.ActivityThread.access$2000(ActivityThread.java:247)
        at android.app.ActivityThread$H.handleMessage(ActivityThread.java:2095)
        at android.os.Handler.dispatchMessage(Handler.java:106)
        at android.os.Looper.loopOnce(Looper.java:201)
        at android.os.Looper.loop(Looper.java:288)
        at android.app.ActivityThread.main(ActivityThread.java:7842)
        at java.lang.reflect.Method.invoke(Native Method)
        at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:548)
        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1003)
     Caused by: java.lang.IllegalArgumentException: com.prismea.android.dev: Targeting S+ (version 31 and above) requires that one of FLAG_IMMUTABLE or FLAG_MUTABLE be specified when creating a PendingIntent.
    Strongly consider using FLAG_IMMUTABLE, only use FLAG_MUTABLE if some functionality depends on the PendingIntent being mutable, e.g. if it needs to be used with inline replies or bubbles.
        at android.app.PendingIntent.checkFlags(PendingIntent.java:375)
        at android.app.PendingIntent.buildServicePendingIntent(PendingIntent.java:724)
        at android.app.PendingIntent.getService(PendingIntent.java:686)
        at io.snapcall.snapcall_android_framework.CallHandler.c(Unknown Source:51)
        at io.snapcall.snapcall_android_framework.CallHandler.onStartCommand(Unknown Source:83)
        at io.snapcall.snapcall_android_framework.VertoSignalingHandler.onStartCommand(Unknown Source:0)
        at android.app.ActivityThread.handleServiceArgs(ActivityThread.java:4643)
```

• Missing `exported` flag
https://developer.android.com/about/versions/12/behavior-changes-12#exported
```xml
        <activity
            android:name="io.snapcall.snapcall_android_framework.CallActivity"
            android:autoRemoveFromRecents="true"
            android:configChanges="screenSize|smallestScreenSize|screenLayout|orientation"
            android:launchMode="singleTop"
            android:screenOrientation="portrait"
            android:supportsPictureInPicture="true" >
            <intent-filter>
                <action android:name=".Snapcall_CallUI_Handler" />

                <category android:name="android.intent.category.DEFAULT" />
            </intent-filter>
        </activity>
```

**Comment by pnoyelle** [comment]
Thanks for the report, i check that.

**Comment by pnoyelle** [comment]
@rlatapy-luna we are checking the change and will release soon.
I will ping you when it's available. 

Thanks for your feedback.

**Comment by pnoyelle** [comment]
Hello @rlatapy-luna should be fixed in lattest release 2.10.1.
